app/service/file_svc.py

```diff
@@ -97,15 +97,21 @@ def __init__(self, plugins, exfil_dir):
         self.special_payloads[name] = func
 
     @staticmethod
-    async def compile_go(platform, output, src_fle):
+    async def compile_go(platform, output, src_fle, ldflags=None):
         """
         Dynamically compile a go file
         :param platform:
         :param output:
         :param src_fle:
+        :param ldflags: A list of string arguments for the golang linker
         :return:
         """
-        os.system('GOOS=%s go build -o %s -ldflags="-s -w" %s' % (platform, output, src_fle))
+        if ldflags:
+            ldflags = '-s -w ' + ' '.join(ldflags)
```

Comment:
can you make the default ldflags='-s -w' ?

then you can overwrite this if desired, which will eliminate the if/else block here.

Reply:
lol yup -- see my comment above haha.